### DISCUSSION TITLE
Seq integration with Azure Monitor's Event Hub

### DISCUSSION THREAD

**Discussion by TysonMN** [datalust/seq-tickets#1547] [Help]
I have an Azure App Service where I run a docker container, and I would like the container logs sent to my Seq server.  I have tried many things and wrote [this detailed StackOverflow question](https://stackoverflow.com/questions/70949986/how-to-send-docker-container-logs-from-azure-app-service-to-seq-log-server) about it, but I haven't been able to find a solution.

I was just reading about Azure Monitor and how it can show me my container logs.  Then I read that the [logs in Azure Monitor can be sent elsewhere using an Event Hub integration](https://docs.microsoft.com/en-us/azure/azure-monitor/partners#partner-tools-with-event-hubs-integration).  I was hoping that Seq would be on that list of existing Event Hub integrations, but it is not.

What would it take to make an Event Hub integration for Seq?

**Comment by nblumhardt** [comment]
Hi Tyson, thanks for the heads-up.

The [_Seq.Input.AzureEventHub_ input](https://github.com/Hinni/Seq.Input.AzureEventHub) looks like it might be close to what you need, though my guess is it'd need some modification to deal with the JSON format used by Azure App Service. If you're able to try it out and figure out where the gaps are, perhaps we can figure out a plan from there?

Best regards,
Nick
